src/jetson/video_capturer.py

```diff
@@ -45,6 +45,8 @@ def __init__(self, gstreamer, dev=0):
             self.capture = cv2.VideoCapture(dev)
 
         _, self.frame = self.capture.read()
+        self.gstreamer = gstreamer
+        self.dev = dev
```

Comment:
what is this for?

Reply:
I needed to make device and gstreamer instance variables so the reboot function can know where to look for a camera